express-notes-api/index.js

```diff
@@ -0,0 +1,160 @@
+const express = require('express');
+const app = express();
+const fs = require('fs');
+
+app.get('/api/notes', (req, res) => {
+  const gradesArray = [];
+  fs.readFile('data.json', 'utf8', (err, data) => {
+    data = JSON.parse(data);
+    const objArray = [];
+    for (const prop in data.notes) {
+      objArray.push({
+        "content": data.notes[prop].content,
+        "id": data.notes[prop].id
+      })
+    }
+    res.json(objArray)
+  })
+});
+
+app.get('/api/notes/:id', (req, res) => {
+  if(req.params.id <= 0) {
+    console.log('gang gang')
+    res.status(400)
+    res.json({"error": "id must be a positive integer"})
```

Comment:
What happens if `id` is either A) not a number, or B) a positive fractional number (ex: `4.32`)? In these cases Express should still send back a `400 Bad Request`.